test/e2e/common/node/runtimeclass.go

```diff
@@ -71,19 +80,79 @@ var _ = SIGDescribe("RuntimeClass", func() {
 	})
 
 	// This test requires that the PreconfiguredRuntimeHandler has already been set up on nodes.
+	// The test CANNOT be made a Conformance as it depends on a container runtime to have a specific handler installed and working.
 	ginkgo.It("should run a Pod requesting a RuntimeClass with a configured handler [NodeFeature:RuntimeHandler]", func() {
 		// Requires special setup of test-handler which is only done in GCE kube-up environment
 		// see https://github.com/kubernetes/kubernetes/blob/eb729620c522753bc7ae61fc2c7b7ea19d4aad2f/cluster/gce/gci/configure-helper.sh#L3069-L3076
 		e2eskipper.SkipUnlessProviderIs("gce")
 
-		rcName := createRuntimeClass(f, "preconfigured-handler", e2enode.PreconfiguredRuntimeClassHandler)
+		rcName := createRuntimeClass(f, "preconfigured-handler", e2enode.PreconfiguredRuntimeClassHandler, nil)
 		defer deleteRuntimeClass(f, rcName)
 		pod := f.PodClient().Create(e2enode.NewRuntimeClassPod(rcName))
 		expectPodSuccess(f, pod)
 	})
 
+	/*
+		Release: v1.20
+		Testname: Can schedule a pod requesting existing RuntimeClass.
+		Description: The Pod requesting the existing RuntimeClass must be scheduled.
+	*/
+	//TODO: Make it conformance
+	ginkgo.It("should schedule a Pod requesting a RuntimeClass without PodOverhead [NodeFeature:RuntimeHandler]", func() {
```

Comment:
this basically a copy of a previous test, except we only test for scheduling of the pod, not successful run as otherwise it cannot be made conformance.

Reply:
added a note to the test description